core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/model/AbstractCamelModelElement.java

```diff
@@ -658,6 +658,10 @@ protected void setParameter(String name, Object value, boolean overrideChangeChe
 		String javaType = getJavaType(name);
 
 		if (this instanceof CamelContextElement) kind = "attribute";
+
+		// this is needed for FUSETOOLS-1884, otherwise some global config
+		// elements lose their children and get corrupted
+		if (getEipByName(CamelUtils.getTranslatedNodeName(getXmlNode())) == null && name.equalsIgnoreCase("id")) kind="attribute";
```

Comment:
basically because the id parameter is set in any case - even without the parseNode